packages/components/src/components/textarea/textarea.css

```diff
@@ -38,11 +38,11 @@ scale-textarea {
   --color-meta-error: var(--telekom-color-text-and-icon-functional-danger);
 
   /*control*/
-  --spacing-control: var(--telekom-spacing-unit-x6) var(--spacing-x)
-    var(--telekom-spacing-unit-x3) calc(var(--spacing-x) - 1px);
+  --spacing-control: 0 var(--spacing-x) var(--spacing-x);
   --transition-control: var(--transition);
   --font-size-control: var(--telekom-typography-font-size-body);
   --background-control: var(--telekom-color-ui-base);
+  --margin-control: var(--spacing-x);
```

Comment:
I noticed that this --margin-control is used for vertical positioning of the label, perhaps we should use `--margin-control: var(--telekom-spacing-unit-x3);` directly to avoid confusion?